Wikipedia/Code/PageHistoryFilterCountsViewController.swift

```diff
@@ -29,7 +29,7 @@ class PageHistoryFilterCountsView: UIView {
                 counts.append(Count(title: WMFLocalizedString("page-history-logged-in-edits", value: "logged-in", comment: "Text for view that shows many edits were made by logged-in users"), image: UIImage(named: "user-edit"), count: loggedInEdits))
             }
             if let unregisteredEdits = editCounts[.customUnregistered]?.count {
-                counts.append(Count(title: WMFLocalizedString("page-history-unregistered-edits", value: "unregistered", comment: "Text for view that shows many edits were made by unregistered users"), image: WMFIcon.temp, count: unregisteredEdits))
+                counts.append(Count(title: WMFLocalizedString("page-history-unregistered-edits", value: "unregistered", comment: "Text for view that shows many edits were made by unregistered users"), image: UIImage(named: "user-anonymous")?.withTintColor(theme.colors.icon ?? .wmf_blue_300, renderingMode: .alwaysTemplate), count: unregisteredEdits))
```

Comment:
A couple of things here:

1. You can flag the asset as template via the asset catalog: 
<img width="1208" alt="Screenshot 2025-07-02 at 4 57 24 PM" src="https://github.com/user-attachments/assets/8434c7e4-8448-46c9-9256-5905ef4883d2" />
2. Then you can replace this with your WMFIcon.anonymous property that you created above:

```
counts.append(Count(title: WMFLocalizedString("page-history-unregistered-edits", value: "unregistered", comment: "Text for view that shows many edits were made by unregistered users"), image: WMFIcon.anonymous, count: unregisteredEdits))
```